tests/unit_tests/test_wave_06.py

```diff
@@ -67,61 +64,47 @@ def test_best_by_category_no_matches_is_none():
 
     assert best_item is None
 
-@pytest.mark.skip
 def test_best_by_category_with_duplicates():
-    # Arrange
     item_a = Clothing(condition=2.0)
     item_b = Clothing(condition=4.0)
     item_c = Clothing(condition=4.0)
     tai = Vendor(
         inventory=[item_a, item_b, item_c]
     )
 
-    # Act
     best_item = tai.get_best_by_category("Clothing")
 
-    # Assert
     assert best_item.get_category() == "Clothing"
     assert best_item.condition == pytest.approx(4.0)
 
-@pytest.mark.skip
 def test_swap_best_by_category():
-    # Arrange
-    # me
     item_a = Decor(condition=2.0)
     item_b = Electronics(condition=4.0)
     item_c = Decor(condition=4.0)
     tai = Vendor(
         inventory=[item_a, item_b, item_c]
     )
 
-    # them
     item_d = Clothing(condition=2.0)
     item_e = Decor(condition=4.0)
     item_f = Clothing(condition=4.0)
     jesse = Vendor(
         inventory=[item_d, item_e, item_f]
     )
 
-    # Act
     result = tai.swap_best_by_category(
         other_vendor=jesse,
         my_priority="Clothing",
         their_priority="Decor"
     )
-
-    raise Exception("Complete this test according to comments below.")
+    assert result == True
+    assert tai.inventory == [item_a, item_b, item_f]
+    assert jesse.inventory == [item_d, item_e, item_c]
```

Comment:
I would prefer to do 3 separate in checks for each item (same comment applies to the assertion statements for the other tests you two completed in this test file). There was no requirement as to where swapped items end up in the lists, and a test that enforces a particular ordering could be thought of as making too many assumptions about the internal implementation of the swapping function, and could prevent future changes to the implementation.

Could also add these assertions to confirm that the list is the right size and that there were no other elements erroneously added to the list (like if the length was 4):
```python
assert len(tai.inventory) == 3
assert len(jesse.inventory) == 3